src/genai/schemas/responses.py

```diff
@@ -47,9 +48,23 @@ class GenAiResponseModel(BaseModel, extra=Extra.allow):
     _extra_fields_warning = root_validator(allow_reuse=True)(alert_extra_fields_validator)
 
 
-class TermsOfUse(GenAiResponseModel):
+class TermsOfUseResult(GenAiResponseModel):
     tou_accepted: bool
     tou_accepted_at: datetime
+    firstName: str
+    lastName: str
+    data_usage_consent: bool = False
+    generate_default: dict = None
+
+
+class TermsOfUse(GenAiResponseModel):
+    results: TermsOfUseResult
+
+
+class GenerateRequestBody(GenAiResponseModel):
+    model_id: str
+    inputs: list[str]
+    parameters: GenerateParams
```

Comment:
If this and `TokenizeRequestBody` are not being used in the core code (and being used only in the extension), then let us move these definitions to the local server extension unless there's a detail being missed here.

Reply:
Good shout, they're not specific to the localserver, but as they're currently only used by it I think that makes sense. 

They're now moved out into `genai/extensions/localserver/schemas.py`